### PR TITLE
[next] Dutch locale for stream

### PR DESCRIPTION
**Pull request by bderen**
## What does this PR do?

Dutch translation for stream

## How do I test this PR?

set in .env LOCALE=nl-NL
check the stream

### CODE REVIEW

**Comment by kgardnr** [comment]
This is great! Thank you @sgtram! 🎉

**Review by kgardnr — APPROVED** [review]

File: src/locales/nl-NL/stream.ftl [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+
+comments-viewNew =
+  { $count ->
+    [1] Nieuwe reactie {$count}
+    *[other] Nieuwe reactie {$count}
+  }
+comments-loadMore = Meer laden
+
+comments-permalinkPopover =
+  .description = A dialog showing a permalink to the comment
```
Comment (wyattjoh) [inline src/locales/nl-NL/stream.ftl]: Did you forget this line? English is already the fallback language.

**Review by wyattjoh — DISMISSED** [review]

File: src/locales/nl-NL/stream.ftl (hunk 1 above)
Comment (bderen) [reply src/locales/nl-NL/stream.ftl]: fixed

**Comment by kgardnr** [comment]
Thank you again @sgtram!

**Pull request merged**